indigo_app/templates/indigo_api/_work_filter_form.html

```diff
@@ -60,9 +35,173 @@
+              </select>
+            </div>
+            <div class="col">
+              <div class="input-group input-daterange {% if form.publication.value != 'range'%}d-none{% endif %}" id="publication-daterange">
+                <input type="text" class="form-control mr-1" data-provide="datepicker" placeholder="start date" pattern="\d{4}-\d\d-\d\d" id="publication_date_start" name="publication_date_start" value="{{ form.publication_date_start.value | default:'' }}">
+                <div class="input-group-addon"> and </div>
+                <input type="text" class="form-control ml-1 mr-1" data-provide="datepicker" placeholder="end date" pattern="\d{4}-\d\d-\d\d" id="publication_date_end" name="publication_date_end" value="{{ form.publication_date_end.value | default:'' }}">
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>    
+
+    <div class="row">
+      <!-- Repealed filter -->
+      <div class="col-md-6 mt-2">
+        <div class="form-group">
+          <label for="{{ form.repeal.id_for_label }}">Repeal</label>
+          <div class="form-row">
+            <div class="col">
+              <select name="{{ form.repeal.html_name }}" class="form-control" id="{{ form.repeal.id_for_label }}" data-toggle="daterange">
+                {% for opt in form.repeal %}
+                  {{ opt }}
+                {% endfor %}
+              </select>
+            </div>
+            <div class="col">
+              <div class="input-group input-daterange {% if form.repeal.value != 'range'%}d-none{% endif %}" id="repeal-daterange">
+                <input type="text" class="form-control mr-1" data-provide="datepicker" placeholder="start date" pattern="\d{4}-\d\d-\d\d" id="repealed_date_start" name="repealed_date_start" value="{{ form.repealed_date_start.value | default:'' }}">
+                <div class="input-group-addon"> and </div>
+                <input type="text" class="form-control ml-1 mr-1" data-provide="datepicker" placeholder="end date" pattern="\d{4}-\d\d-\d\d" id="repealed_date_end" name="repealed_date_end" value="{{ form.repealed_date_end.value | default:'' }}">
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- Commenced filter -->
+      <div class="col-md-6 mt-2">
+        <div class="form-group">
+          <label for="{{ form.commencement.id_for_label }}">Commencement</label>
+          <div class="form-row">
+            <div class="col">
+              <select name="{{ form.commencement.html_name }}" class="form-control" id="{{ form.commencement.id_for_label }}" data-toggle="daterange">
+                {% for opt in form.commencement %}
+                  {{ opt }}
+                {% endfor %}
+              </select>
+            </div>
+            <div class="col">
+              <div class="input-group input-daterange {% if form.commencement.value != 'range'%}d-none{% endif %}" id="commencedment-daterange">
```

Comment:
```suggestion
              <div class="input-group input-daterange {% if form.commencement.value != 'range'%}d-none{% endif %}" id="commencement-daterange">
```

Reply:
I had just spotted it and was about to commit😅